wormhole-connect/src/views/Bridge/Inputs/From.tsx

```diff
@@ -187,14 +187,16 @@ function FromInputs() {
         balance={balance}
         tokenPrice={getTokenPrice(prices, config.tokens[token])}
       />
-      <TokensModal
```

Comment:
rendering this modal when it's not even showing was sometimes causing errors in builder when switching between testnet and mainnet due to a race condition.

i think it's faster and simpler to just not render it (and run all of its code) in the default, zero-state view of Connect.